package.json

```diff
@@ -27,7 +27,9 @@
     "build:all": "nx run-many --target=build",
     "build": "nx affected --target=build --parallel=3",
     "build:non-demo": "nx affected --target=build --parallel=3 --exclude=@rango-dev/queue-manager-demo,@rango-dev/wallets-demo,@rango-dev/wallets-adapter-demo,@rango-dev/widget-app,@rango-dev/widget-playground",
-    "dev:ui": "nx run-many --target=storybook --projects=@rango-dev/ui",
+    "dev:ui:type-checker": "node ./scripts/dev-ts-check/command.mjs --project @rango-dev/storybook",
+    "dev:ui:dev-server": "yarn --cwd ./widget/storybook/ dev",
+    "dev:ui": "npm-run-all --parallel dev:ui:dev-server dev:ui:type-checker",
```

Comment:
please use this command instead and remove the other ones as well:
```
"dev:ui": "nx run @rango-dev/storybook:dev",
```

Running using nx will build necessary packages as well.